prog/mutation.go

```diff
@@ -102,9 +102,16 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable) {
 						filename := r.filename(s)
 						arg.Data = []byte(filename)
 					case sys.ArrayType:
-						count := r.rand(6)
-						if count == uintptr(len(arg.Inner)) {
```

Comment:
This was useful code. Keep it.
Mutation frequently generated no-op mutation, which caused tests to fail and just reduced fuzzing efficiency. You can see retry flag at the top of the function, which is another piece of logic that ensures that mutation actually changes the program.